terraform-environments/aws/dev/helm/70-gem-frontend/helm_values.yaml

```diff
@@ -33,7 +33,7 @@ ingress:
     # kubernetes.io/tls-acme: "true"
   paths:
     - path: /
-      servicePort: http
+      servicePort: 80
```

Comment:
Just checking this isn't a typo

Reply:
yes the port number here needs to line up with the service definition above